src/data/drinkRecipes/spicedAppleFizz.ts

```diff
@@ -0,0 +1,71 @@
+import type { DrinkRecipe } from "types/DrinkRecipe"
+import { DrinkRecipeIngredientType } from "types/DrinkRecipe"
+
+const fakeImageUrl =
+  "https://www.liquor.com/thmb/0Q9zUdEgdxqDdU9aqwhVbmcsz0Q=/960x0/filters:no_upscale():max_bytes(150000):strip_icc()/spiced-apple-fizz-720x720-primary-0bbfd940b7fb473284344b2ae591864a.jpg"
+
+const spicedAppleFizz: DrinkRecipe = {
+  name: "Spiced Apple Fizz",
+  imageUrl: fakeImageUrl,
+  measuredIngredients: [
+    {
+      type: DrinkRecipeIngredientType.Measured,
+      ingredient: "Vodka",
+      unit: "oz",
+      unitAmount: 1.5,
+      alternates: [],
+    },
+    {
+      type: DrinkRecipeIngredientType.Measured,
+      ingredient: "Pear Brandy",
+      unit: "oz",
+      unitAmount: 0.5,
+      alternates: [],
+    },
+    {
+      type: DrinkRecipeIngredientType.Measured,
+      ingredient: "Lemon Juice",
+      unit: "oz",
+      unitAmount: 0.5,
+      alternates: [],
+    },
+    {
+      type: DrinkRecipeIngredientType.Measured,
+      ingredient: "Honey Syrup",
+      unit: "oz",
+      unitAmount: 0.5,
+      alternates: [],
+    },
+  ],
+  numberedIngredients: [],
+  portionlessIngredients: [
+    {
+      type: DrinkRecipeIngredientType.Portionless,
+      ingredient: "Hard Apple Cider",
+      alternates: [],
+    },
+  ],
+  garnishIngredients: [
+    {
+      type: DrinkRecipeIngredientType.Portionless,
+      ingredient: "Apple",
+      suffix: "Fan",
+      alternates: [],
+    },
+    {
+      type: DrinkRecipeIngredientType.Portionless,
+      prefix: "Grated",
+      ingredient: "Cinnamon",
+      alternates: [],
```

Comment:
TBF since I criticized your usage above, I like the usage of the prefix here for cinnamon and the suffix for apple. It's a simple prep step that doesn't actually change the ingredient you start with / buy at the store. You don't buy an apple fan; you buy an apple and then you fan it yourself, so I think that's great. While that might be less true for cinnamon (maybe you can buy that grated), that usage serves to just keep the ingredient/alias lists (and therefor search time) shorter, and anyone can grate their cinnamon.